tiatoolbox/tools/patchextraction.py

```diff
@@ -53,29 +52,86 @@ class PatchExtractor(ABC):
         input_img(ndarray, WSIReader): input image for patch extraction.
           input_image type is ndarray for an image tile whereas :obj:`WSIReader`
           for an WSI.
-        patch_size(Tuple of int): patch size tuple (width, height).
-        resolution(Tuple of int): resolution at which to read the image.
+        patch_size(tuple(int)): patch size tuple (width, height).
+        resolution(tuple(int)): resolution at which to read the image.
         units (str): the units of resolution.
         n(int): current state of the iterator.
+        locations_df(pd.DataFrame): A table containing location and/or type of patch.
 
     """
 
     def __init__(self, input_img, patch_size, resolution=0, units="level"):
-        self.patch_size = patch_size
+        if isinstance(patch_size, (tuple, list)):
+            self.patch_size = (int(patch_size[0]), int(patch_size[1]))
+        else:
+            self.patch_size = (int(patch_size), int(patch_size))
```

Comment:
Highligh to refer back to above comment.

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/97/commits/f85a61e9650b7dad8ffa73f6eafd37c5d18e5807